calculator-tests/compute_tests.m

```diff
@@ -0,0 +1,41 @@
+//
+//  compute_tests.m
+//  calculator-tests
+//
+//  Created by blakelucchesi on 7/24/20.
+//  Copyright © 2020 nonya. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import "compute.h"
+
+@interface compute_tests : XCTestCase
+
+- (void)assert:(char *)a equals:(char *)b;
+
+@end
+
+@implementation compute_tests
+
+- (void)assert:(char *)a equals:(char *)b {
```

Comment:
This is some quick code that checks each byte of the `char *` inputs to assert whether or not the two strings are equal. Equality is a whole other topic that we can work through later.